WebApi/Services/CreateReviewsJobProcessor.cs

```diff
@@ -15,20 +15,17 @@ public CreateReviewsJobProcessor(IConfiguration configuration, AppDbContext dbCo
         _dbContext = dbContext;
     }
 
-    public async Task<string?> ProcessJobAsync(Job job, CancellationToken cancellationToken)
+    public async Task<string?> ProcessJobAsync(Job job, CancellationToken stoppingToken)
     {
         var programCommitteePrivateKey = KeyUtils.GetProgramCommitteePrivateKey();
         var submissionId = Guid.Parse(job.Payload!);
-
-        var submissionsDirectoryPath = _configuration.GetSubmissionsStorage();
-        var paperFilePath = Path.Combine(submissionsDirectoryPath, submissionId.ToString());
-        var paperBytes = await File.ReadAllBytesAsync(paperFilePath);
+        var paperBytes = await GetPaperAsync(submissionId);
 
         var reviewers = _dbContext.Reviewers.AsAsyncEnumerable();
 
         await foreach (var reviewer in reviewers)
         {
-            cancellationToken.ThrowIfCancellationRequested();
+            stoppingToken.ThrowIfCancellationRequested();
```

Comment:
Likewise here: Method is called "ThrowIfCancellationRequested". Can't see why you chose to rename it from cancellationToken to stoppingToken

Reply:
Read the documentation of BackgroundService.